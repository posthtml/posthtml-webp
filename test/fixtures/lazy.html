<!doctype html>
<html>
    <body>
        <img data-src="photo.jpg">
        <img data-src="photo.jpg" src="low-quality-preview.jpg">
        <img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w">
        <img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w" src="low-quality-preview.jpg">
        <img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w" data-src="photo.jpg">
        <img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w" data-src="photo.jpg" src="low-quality-preview.jpg">
    </body>
</html>
