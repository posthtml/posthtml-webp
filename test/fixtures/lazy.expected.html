<!doctype html>
<html>
    <body>
        <picture><source type="image/webp" data-srcset="photo.webp"><img data-src="photo.jpg"></picture>
        <picture><source type="image/webp" data-srcset="photo.webp"><img data-src="photo.jpg" src="low-quality-preview.jpg"></picture>
        <picture><source type="image/webp" data-srcset="photo-lg.webp 1000w, photo-md.webp 500w, photo-sm.webp 250w"><img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w"></picture>
        <picture><source type="image/webp" data-srcset="photo-lg.webp 1000w, photo-md.webp 500w, photo-sm.webp 250w"><img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w" src="low-quality-preview.jpg"></picture>
        <picture><source type="image/webp" data-srcset="photo-lg.webp 1000w, photo-md.webp 500w, photo-sm.webp 250w"><img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w" data-src="photo.jpg"></picture>
        <picture><source type="image/webp" data-srcset="photo-lg.webp 1000w, photo-md.webp 500w, photo-sm.webp 250w"><img data-srcset="photo-lg.jpg 1000w, photo-md.jpg 500w, photo-sm.jpg 250w" data-src="photo.jpg" src="low-quality-preview.jpg"></picture>
    </body>
</html>
